driver.go

```diff
@@ -723,6 +723,7 @@ func checkIsValidType(v driver.Value) bool {
 	default:
 		return false
 	case nil:
+	case uint:
```

Comment:
nit: move this down to line 741 and also include support for `[]uint` and `*uint` if possible.